src/components/Emoji.vue

```diff
@@ -126,6 +126,10 @@ export default {
     height: {
       type: String,
       default: "250px"
+    },
+    searchLabel: {
+      type: String,
+      default: "Search"
```

Comment:
@vestusola  i think i missed something here, this seems to have broken the app

Reply:
Oops!